velox/functions/sparksql/aggregates/tests/FirstAggregateTest.cpp

```diff
@@ -241,6 +241,74 @@ TEST_F(FirstAggregateTest, dateGlobal) {
   testGlobalAggregate(vectors, ignoreNullData, hasNullData);
 }
 
+TEST_F(FirstAggregateTest, shortDecimalGroupBy) {
+  auto vectors = {makeRowVector({
+      makeFlatVector<int32_t>(4, [](auto row) { return row % 2; }),
+      makeNullableFlatVector<int64_t>(
```

Comment:
nit: should we include `decimal::min`, `decimal::max` values to cover edges cases?

Reply:
hmm, these functions does not include decimal calculate logic, I think current tests already enough, hope that reason make sense for you.